selenium_containers.py

```diff
@@ -0,0 +1,26 @@
+import os
+import docker
+
+
+def start_firefox():
+
+    container_time_zone = os.getenv('CONTAINERTZ', "TZ=US/Mountain")
+
+    client = docker.from_env()
+    client.containers.run(
+        "selenium/standalone-firefox:4.0.0-alpha-7-prerelease-20200826",
```

Comment:
Is there a way to specify this without an explicit version so that it just gets the latest stable release (I would prefer not to get prereleases)? Maybe something like `selenium/standalone-firefox:latest`? If not, we need to know to go to https://github.com/SeleniumHQ/docker-selenium/releases to get the latest release.